### DISCUSSION TITLE
documentation for using external postgres db and chroma db

### DISCUSSION THREAD

**Discussion by santosh0787** [open-webui/open-webui#1957] [Q&A]
Hello,
As mentioned in release notes for 0.1.122, can you provide documentation on how set env variable for Chroma db servers? 
release notes: 
🛢️ External Database Support: Seamlessly connect to custom SQLite or Postgres databases using the 'DATABASE_URL' environment variable.
🌐 Remote ChromaDB Support: Introducing the capability to connect to remote ChromaDB servers.

**Comment by RWayne93** [comment]
Or PGvector would be nice as well.

**Comment by Hugobox** [comment]
Also interested in knowing more how to connect to remote ChromaDB

**Reply by justinh-rahb** [reply]
## ChromaDB Environment Variables

### `CHROMA_TENANT`
**Default**: `default_tenant`
**Description**: Sets the tenant for ChromaDB to use for RAG embeddings.

### `CHROMA_DATABASE`
**Default**: `default_database`
**Description**: Sets the database in the ChromaDB tenant to use for RAG embeddings.

### `CHROMA_HTTP_HOST`
**Description**: Specifies the hostname of a remote ChromaDB Server. Uses a local ChromaDB instance if not set.

### `CHROMA_HTTP_PORT`
**Default**: `8000`
**Description**: Specifies the port of a remote ChromaDB Server.

### `CHROMA_HTTP_HEADERS`
**Description**: Comma-separated list of HTTP headers to include with every ChromaDB request.
**Example**: `Authorization=Bearer heuhagfuahefj,User-Agent=OpenWebUI`.

### `CHROMA_HTTP_SSL`
**Default**: `False`
**Description**: Controls whether or not SSL is used for ChromaDB Server connections.
